clj-lex.el

```diff
@@ -167,13 +202,24 @@ behavior."
 (defun clj-lex-keyword ()
   (let ((pos (point)))
     (right-char)
-    (when (equal (char-after (point)) ?:)
+    (when (equal (char-after (point)) ?:) ;; same-namespace keyword
       (right-char))
-    (if (clj-lex-symbol-start? (char-after (point)))
-        (clj-lex-token :keyword (clj-lex-get-symbol-at-point pos) pos)
+    (if (equal (char-after (point)) ?:) ;; three colons in a row => lex-error
+        (progn
+          (right-char)
+          (clj-lex-token :lex-error (buffer-substring-no-properties pos (point)) pos 'error-type :invalid-keyword))
       (progn
-        (right-char)
-        (clj-lex-token :lex-error (buffer-substring-no-properties pos (point)) pos 'error-type :invalid-keyword)))))
+        (while (or (clj-lex-symbol-rest? (char-after (point)))
+                   (equal (char-after (point)) ?#))
```

Comment:
Playing around with `#` I'm noticing it can also be part of symbols and tagged literals:

```clojure
user=> (let [a#b 1] a#b)     
1
user=> (let [#a#b 1] #a#b)
RuntimeException No reader function for tag a#b  clojure.lang.LispReader$CtorReader.readTagged (LispReader.java:1245)
```

So maybe we could add this equality check to `clj-lex-symbol-rest?`